addons/website_blog/static/src/js/website_blog.inline.discussion.js

```diff
@@ -50,6 +51,21 @@ var InlineDiscussion = core.Class.extend({
                         'marginLeft': "+=250px"
                     });
                 }
+            } else if ($(event.target).hasClass('discussion-link')) {
```

Comment:
please concatenate the two conditions, the content is the same (the same except: self.hide_discussion() but self.hide_discussion() is call latter with the check on active class name)